modules/network-observability-filtering-ebpf-rule.adoc

```diff
@@ -29,22 +34,31 @@ spec:
   agent:
     type: eBPF
     ebpf:
+      privileged: true <1>
+      features:
+        - PacketDrop
       flowFilter:
-        action: Accept  <1>
-        cidr: 172.210.150.1/24 <2>
-        protocol: SCTP
-        direction: Ingress
-        destPortRange: 80-100
-        peerIP: 10.10.10.10
-        enable: true    <3>
+        enable: true <2>
+        rules:
+          - action: Accept  <3>
+            cidr: 172.210.150.1/24 <4>
+            protocol: SCTP
+            direction: Ingress
+            destPortRange: 80-100
+            peerIP: 10.10.10.10
+          - action: Accept
+            cidr: 172.30.0.0/16
+            pktDrops: true <5>
 ----
-<1> The required `action` parameter describes the action that is taken for the flow filter rule. Possible values are `Accept` or `Reject`. 
-<2> The required `cidr` parameter provides the IP address and CIDR mask for the flow filter rule and supports IPv4 and IPv6 address formats. If you want to match against any IP address, you can use `0.0.0.0/0` for IPv4 or `::/0` for IPv6.
-<3> You must set `spec.agent.ebpf.flowFilter.enable` to `true` to enable this feature.
+<1> You must set `spec.agent.ebpf.privileged` to `true` for the packet drop feature.
+<2> You must set `spec.agent.ebpf.flowFilter.enable` to `true` to enable eBPF flow filtering.
+<3> The required `action` parameter describes the action that is taken for the flow filter rule. Possible values are `Accept` or `Reject`.
+<4> The required `cidr` parameter provides the IP address and CIDR mask for the flow filter rule and supports IPv4 and IPv6 address formats. If you want to match against any IP address, you can use `0.0.0.0/0` for IPv4 or `::/0` for IPv6.
+<5> By default, all other flows are rejected.
 ====
 +
 [%collapsible]
-.See flows to any addresses outside the cluster 
+.See flows to any addresses outside the cluster
```

Comment:
And instead of this use `Filter flows with packet drops` example in the same blog